tests/SDK/Util/SpanData.php

```diff
@@ -10,20 +10,23 @@
 use OpenTelemetry\SDK\InstrumentationLibrary;
 use OpenTelemetry\SDK\Resource\ResourceInfo;
 use OpenTelemetry\SDK\Trace as SDK;
+use OpenTelemetry\SDK\Trace\EventInterface;
+use OpenTelemetry\SDK\Trace\LinkInterface;
 use OpenTelemetry\SDK\Trace\StatusData;
 
 class SpanData implements SDK\SpanDataInterface
 {
     /** @var non-empty-string */
     private string $name = 'test-span-data';
 
-    /** @var list<API\EventInterface> */
+    /** @var list<EventInterface> */
     private array $events = [];
 
-    /** @var list<API\LinkInterface> */
+    /** @var list<LinkInterface>
+     */
     private array $links = [];
 
-    private API\AttributesInterface $attributes;
+    private \OpenTelemetry\SDK\AttributesInterface $attributes;
```

Comment:
I think the project style is to have these as `use` statements

Reply:
Missed this one, PhpStorm changed some use-statements to fqcn on moving these classes.